src/array_of_fixedsize.jl

```diff
@@ -12,11 +12,15 @@ call(::MaxFun, a, b) = max(a, b)
 call(::MinFun, a, b) = min(a, b)
 minimum{T <: FixedArray}(a::Vector{T}) = reduce(MinFun(), a)
 maximum{T <: FixedArray}(a::Vector{T}) = reduce(MaxFun(), a)
-function isapprox{FSA <: FixedArray, A <: Union{Array, FixedArray}}(a::FSA, b::A)
-    for i=1:length(a)
-        !isapprox(a[i], b[i]) && return false
-    end
-    true
+
+function isapprox{FSA <: FixedArray, A <: FixedArray}(x::FSA, y::A; rtol::Real=Base.rtoldefault(eltype(x),eltype(y)), atol::Real=0, norm::Function=vecnorm)
+    # Same behaviour as AbstractArray: julia/base/linalg/generic.jl
+	d = norm(x - y)
+    return isfinite(d) ? d <= atol + rtol*max(norm(x), norm(y)) : x == y
+end
+function isapprox{FSA <: FixedArray, A <: Array}(x::FSA, y::A; rtol::Real=Base.rtoldefault(eltype(x),eltype(y)), atol::Real=0, norm::Function=vecnorm)
+    d = norm(Array(x) - y) # operations between Arrays and FixedArrays not defined...
```

Comment:
Heh, I'm just digging in the `map` code now.  No promises I can also fix this one, but I'll see.